man/man1/p11sak.1.in

```diff
@@ -238,11 +238,52 @@ attribute of the key and
 can be used to set the binary attributes of the key (see below for detailed description of the attributes).
 .
 .PP
+.SS "Generating IBM Dilithium keys"
+.
+.B p11sak
+.BR generate-key | gen-key | gen
+.BR ibm-dilithium
+.B \-\-slot
```

Comment:
Maybe one small enhancement to make life easier in the future (and staying backward compatible):

Currently OCK only supports Dilithium keys of the NIST round 2 submission (this corresponds to OID 1.3.6.1.4.1.2.267.1.6.5).
In the future, other Dilithium variants will be supported, i.e. for round 3 submissions and/or different strength settings. So maybe we should already now add a keyword for the key generation to specify the Dilithium variant to generate. Currently we would only support one variant though. 

Not sure how to specify the variant, specifying the OID is probably not very user friendly. Maybe something like "r2_65" (according to round 2 and the last 2 numbers of the OID - .6.5). I guess this will fit best to what's coming up later.

So similar to RSA, where you specify "rsa 1024", you would specify "ibm-dilithium r2_65" to generate a Dilithium key with that variant.

Reply:
done